libs/stats/odc/stats/_gm_ls_bitmask.py

```diff
@@ -35,6 +36,7 @@ def __init__(
             work_chunks: Tuple[int, int] = (400, 400),
             scale: float = 0.0000275,
             offset: float = -0.2,
+            output_scale: int = 10000, # scale USGS Landsat bands into surface reflectance
```

Comment:
Can we please change the comment here to instead read 'making SR range match sentinel-2 gm'